### ISSUE TITLE
A TAB ( \t ) in a text parameter value seems to create corrupt ifc file

### ISSUE THREAD

**Issue by Teun1** [Autodesk/revit-ifc#601]
Revit 2022.1.3 + IFC exporter 22.6.0.0

I ran into a strange bug when I tried to get my IFC in Solibri.
Looks like I had some text parameters with a value containing a tab ( \t )
And the IFC really seems a bit strange.
BIMcollab ZOOM doesn't care, Solibri does.

The parameters are project information parameters.
And we use a CustomPset file to get that into the IFC.

Manually removing the tabs in the IFC solves the problem.
But perhaps this is also undesirable behavior on the part of the exporter.
I have attached some small test files.
For now, I just delete those tabs manually.

![2023-02-22 16_11_41-BIMcollab ZOOM](https://user-images.githubusercontent.com/7262482/220671726-47486792-7609-4328-8844-c3a24bc1869f.png)
![2023-02-22 16_13_29-IFC](https://user-images.githubusercontent.com/7262482/220671730-328904ce-52ba-49a4-a6a4-400a9f3738a5.png)
![2023-02-22 16_10_05-Solibri Office](https://user-images.githubusercontent.com/7262482/220671733-2b156169-6da4-4b74-865c-86795d3694fe.png)
[230222_TAB.zip](https://github.com/Autodesk/revit-ifc/files/10805490/230222_TAB.zip)

**Comment by AngelVelezSosa** [comment]
Can you try this on Revit 2023?  I believe there was a bug in the ODA 22.x line dealing with non-visible characters in text.

**Comment by Teun1** [comment]
In R23 the custom mapping to IfcProject does not work anymore. Not sure why. 
I liked the way I could distinquish between IfcProject, IfcSite and IfcBuilding.
But if I changed the mapping to IfcSite instead. And the tab \t is changed for a space and we got a stable ifc again.
Is there a reason why the mapping to IfcProject does not work in R23 anymore?
[230222_TAB_R23.zip](https://github.com/Autodesk/revit-ifc/files/10807696/230222_TAB_R23.zip)
